damus/Views/Relays/RelayPicView.swift

```diff
@@ -42,19 +42,24 @@ struct InnerRelayPicView: View {
             Color(uiColor: .secondarySystemBackground)
 
             if let url {
-                KFAnimatedImage(url)
-                    .imageContext(.pfp, disable_animation: disable_animation)
-                    .onFailure { _ in
-                        failedImage = true
-                    }
-                    .cancelOnDisappear(true)
-                    .configure { view in
-                        view.framePreloadCount = 3
-                    }
-                    .placeholder { _ in
-                        Placeholder(url: url)
-                    }
-                    .scaledToFit()
+                if url.absoluteString.contains(".onion") {
```

Comment:
Although unlikely, this logic might match other non-onion URLs (for example `wss://relay.onionlovers.com/v2`). We can improve this logic to avoid such mismatches, by using properties such as [the host property in URLComponents](https://developer.apple.com/documentation/foundation/urlcomponents/1779882-host) and [hasSuffix](https://developer.apple.com/documentation/swift/string/hassuffix(_:)), for example.